.github/workflows/test.yml

```diff
@@ -53,8 +53,7 @@ jobs:
       - name: Run SageMath doctests
         shell: bash -l {0}
         run: |
-          export SURFACE_DYNAMICS_INSTALL="$(sage -c 'import surface_dynamics; print(surface_dynamics.__path__[0])')"
-          sage -tp --force-lib --long --optional=${{ matrix.optionals }} ${SURFACE_DYNAMICS_INSTALL} doc
+          sage -tp --force-lib --long --optional=${{ matrix.optionals }} surface_dynamics doc
```

Comment:
This turns out to be problematic as depending on the sage version not all source files are installed (namely `misc/generalized_multiple_zeta_values.py` is installed depending whether multiple zeta values are available in sage).

Reply:
See https://github.com/flatsurf/surface-dynamics/runs/4825797004

Reply:
Should be fixed by 7284697